frontend/vendor/Hotjar.js

```diff
@@ -0,0 +1,13 @@
+(function(h, o, t, j, a, r) {
+  h.hj =
+    h.hj ||
+    function() {
+      (h.hj.q = h.hj.q || []).push(arguments);
+    };
+  h._hjSettings = { hjid: 582685, hjsv: 5 };
```

Comment:
I don't think this ID should be hardcoded.

Reply:
Added Hotjar ID environment variable to production server:

![image](https://user-images.githubusercontent.com/893208/40643103-e3e446b0-6316-11e8-8e0e-1c5ed8d7b923.png)